tests/tests_cib_resources_create.yml

```diff
@@ -514,23 +530,29 @@
           when:
             - '"pcmk.resource.config.output-formats" not in __test_pcs_capabilities'
           vars:
+            # wokeignore:rule=master
+            __mstr: 'Master'
+            # wokeignore:rule=slave
+            __slv: 'Slave'
+            # wokeignore:rule=dummy
+            __dmy: 'Dummy'
```

Comment:
I think using `__dmy` here is ok (as opposed to other places where you replaced the string "dummy" with "dmy" - those places should use "test" or "example" or "resource" or similar instead of "dmy") - in this case it is good because it is a replacement for places where you must use the explicit string `Dummy` and it signifies that